otter/integration/lib/autoscale.py

```diff
@@ -259,6 +340,9 @@ def start(self, rcs, test):
         def record_results(resp):
             rcs.groups.append(resp)
             self.group_id = str(resp["group"]["id"])
+            if verbosity > 0:
```

Comment:
Also non-blocking issue: Trial provides support for logging.  We should probably look into using that going forward.